### ISSUE TITLE
Cannot install through APT

### ISSUE THREAD

**Issue by decapstrike** [andresriancho/w3af#16570]
In the most recent version of Kali (2018.1, then dist-upgrade, installed today) neither installation method works. I cannot use APT to install, get the following message:

Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package w3af is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'w3af' has no installation candidate

When using apt-cache search w3af golismero is the package referencing it. When trying to build from source, it fails, manually installing what it's failing on (libssl-dev, libxml2-dev, and libxslt-dev) results in the compilation then starts failing with this message:
. /tmp/w3af_dependency_install.sh
Traceback (most recent call last):
  File "/usr/bin/pip", line 9, in <module>
    from pip import main
  File "/usr/lib/python2.7/dist-packages/pip/__init__.py", line 21, in <module>
    from pip._vendor.requests.packages.urllib3.exceptions import DependencyWarning
  File "/usr/lib/python2.7/dist-packages/pip/_vendor/__init__.py", line 64, in <module>
    vendored("cachecontrol")
  File "/usr/lib/python2.7/dist-packages/pip/_vendor/__init__.py", line 36, in vendored
    __import__(modulename, globals(), locals(), level=0)
  File "/usr/share/python-wheels/CacheControl-0.11.7-py2.py3-none-any.whl/cachecontrol/__init__.py", line 9, in <module>
  File "/usr/share/python-wheels/CacheControl-0.11.7-py2.py3-none-any.whl/cachecontrol/wrapper.py", line 1, in <module>
  File "/usr/share/python-wheels/CacheControl-0.11.7-py2.py3-none-any.whl/cachecontrol/adapter.py", line 4, in <module>
  File "/usr/share/python-wheels/requests-2.12.4-py2.py3-none-any.whl/requests/__init__.py", line 52, in <module>
  File "/usr/share/python-wheels/requests-2.12.4-py2.py3-none-any.whl/requests/packages/__init__.py", line 59, in <module>
  File "/usr/share/python-wheels/requests-2.12.4-py2.py3-none-any.whl/requests/packages/__init__.py", line 32, in vendored
  File "/usr/share/python-wheels/urllib3-1.19.1-py2.py3-none-any.whl/urllib3/contrib/pyopenssl.py", line 47, in <module>
  File "/usr/local/lib/python2.7/dist-packages/OpenSSL/__init__.py", line 8, in <module>
    from OpenSSL import rand, crypto, SSL
  File "/usr/local/lib/python2.7/dist-packages/OpenSSL/SSL.py", line 118, in <module>
    SSL_ST_INIT = _lib.SSL_ST_INIT
AttributeError: 'module' object has no attribute 'SSL_ST_INIT'

I reinstalled Kali initially thinking I had some conflict, but it's persisted on this new installation as well. Thanks.

**Comment by andresriancho** [comment]
Don't try to install from apt. Always use the latest from our repo.

```
git clone ...
cd w3af
./w3af_console
```

**Comment by decapstrike** [comment]
When apt didn't work, I followed the instructions and cloned the repo, after doing the ./w3af_console it correctly the tmp file to build dependencies, and that's where things really started to go wrong. It would start trying to install, find errors saying I was missing certain dependencies, primarily libssl-dev, libxml2-dev, and libxslt-dev. After manually installing them, then the error I copied above appears. And after that, the majority of Python code I have doesn't work correctly. Thought it was just an old dependency fouling it up, so I started a fresh Kali instance, exact same issue. Trying in docker now.
